Gordon360/Controllers/ScheduleController.cs

```diff
@@ -23,7 +24,18 @@ public class ScheduleController(IScheduleService scheduleService) : GordonContro
     [StateYourBusiness(operation = Operation.READ_ONE, resource = Resource.STUDENT_SCHEDULE)]
```

Comment:
I think it would be better to delete this `StateYourBusiness` attribute, rather than leaving it when it doesn't actually enforce any authorization. Otherwise, people might assume it provides some security that it doesn't.